packages/bitcore-wallet-service/src/lib/storage.ts

```diff
@@ -76,6 +76,10 @@ export class Storage {
     db.collection(collections.TXS).createIndex({
       txid: 1
     });
+    db.collection(collections.TXS).createIndex({
+      multisigContractAddress: 1,
```

Comment:
this will be VERY slow to create, can we find an alternative way?

Reply:
Done. I remove it. Not needed.